.github/workflows/agw-workflow.yml

```diff
@@ -0,0 +1,301 @@
+---
+name: agw-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+  pull_request:
+    branches:
+      - master
```

Comment:
This should also be tested on PR to `v*` branches example v1.5 v1.6 branches

Reply:
sure! added.